### PR TITLE
STITCH-1095 Google and Facebook Auth providers broken in Android SDK

### PR DESCRIPTION
**Pull request by jsflax**

### CODE REVIEW

File: stitch/src/main/java/com/mongodb/stitch/android/auth/oauth2/facebook/FacebookAuthProvider.java [hunk 1]
```diff
@@ -1,18 +1,18 @@
 package com.mongodb.stitch.android.auth.oauth2.facebook;
 
 import com.mongodb.stitch.android.auth.AuthProvider;
+import com.mongodb.stitch.android.auth.oauth2.OAuth2;
```
Comment (edaniels) [inline stitch/src/main/java/com/mongodb/stitch/android/auth/oauth2/facebook/FacebookAuthProvider.java]: The changes in this file have been committed before.

File: stitch/src/main/java/com/mongodb/stitch/android/auth/oauth2/google/GoogleAuthProvider.java [hunk 2]
```diff
@@ -1,18 +1,18 @@
 package com.mongodb.stitch.android.auth.oauth2.google;
 
 import com.mongodb.stitch.android.auth.AuthProvider;
+import com.mongodb.stitch.android.auth.oauth2.OAuth2;
```
Comment (edaniels) [inline stitch/src/main/java/com/mongodb/stitch/android/auth/oauth2/google/GoogleAuthProvider.java]: The changes in this file have been committed before.

File: stitch/src/main/java/com/mongodb/stitch/android/auth/oauth2/facebook/FacebookAuthProviderInfo.java [hunk 3]
```diff
@@ -16,7 +16,8 @@
 public class FacebookAuthProviderInfo extends AuthProviderInfo {
     public static final String FQ_NAME = "oauth2-facebook";
 
-    static final String METADATA_FIELDS = "metadata_fields";
```
Comment (edaniels) [inline stitch/src/main/java/com/mongodb/stitch/android/auth/oauth2/facebook/FacebookAuthProviderInfo.java]: It seems like this is the only change you're trying to make in this PR.

**Comment by jsflax** [comment]
Yes, one commit was merged. The change is adding a the metadata_fields to the constructor for Facebook info. I did not want to have to open up another JIRA ticket.

**Comment by edaniels** [comment]
Using the same JIRA ticket is fine, but we only want new commits, not an older branch.

**Pull request merged**